web/crux-ui/src/components/settings/create-token-card.tsx

```diff
@@ -6,15 +6,15 @@ import { DyoHeading } from '@app/elements/dyo-heading'
 import { DyoInput } from '@app/elements/dyo-input'
 import { DyoLabel } from '@app/elements/dyo-label'
 import { defaultApiErrorHandler } from '@app/errors'
-import { GeneratedToken, GenerateToken } from '@app/models'
+import { GeneratedToken, GenerateToken, TokenExpiration } from '@app/models'
 import { API_TOKENS } from '@app/routes'
 import { sendForm } from '@app/utils'
 import { generateTokenSchema } from '@app/validations/token'
 import { useFormik } from 'formik'
 import useTranslation from 'next-translate/useTranslation'
 import { MutableRefObject } from 'react'
 
-const EXPIRATION_VALUES = [30, 60, 90]
+const EXPIRATION_VALUES: TokenExpiration[] = [30, 60, 90, 'never']
```

Comment:
Not better to use 0 for never, and we can skip check like these `typeof value === 'number'` ?

Reply:
I wanted to be explicit about it, but 0 could work too.

Reply:
Mixing strings with numbers not a good approach IMHO.